src/System.Console/tests/System.Console.Tests.csproj

```diff
@@ -27,6 +27,9 @@
     <Compile Include="ThreadSafety.cs" />
     <Compile Include="XunitAssemblyAttributes.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
+    <Compile Include="TermInfo.cs" />
+  </ItemGroup>
```

Comment:
Hmm, I forgot this would be an issue.  Have you verified that the tests are actually running as part of CI?  I don't remember now if we assume the test assemblies are the same for both Windows and Unix.  If this is an issue, you could switch to using reflection to access internals rather than using InternalsVisibleTo.